tests/test_part_one.py

```diff
@@ -0,0 +1,88 @@
+from source.part_one import categorize_strand, decode_strand, encode_strand
+from .test_data import DNA, DNA_ENCODED, RNA, RNA_ENCODED, STRAND_BOTH_BASES, STRAND_NEITHER_BASE
```

Comment:
I have minor concerns that having the values defined elsewhere may make it a bit more difficult for folks to understand the tests (like we see with the data in Viewing Party, though tat's much more complex in general), but it makes sense to do it this way since the values are used in the two (now separated) test files.

Maybe since this data is easier to think about, seeing it presented this way will even help when we get to Viewing Party!